src/content/1.7/basics/installation/_index.md

```diff
@@ -87,27 +87,37 @@ This step is only needed if you downloaded the development version.
 Use [composer](https://getcomposer.org/download/) to download the project's dependencies:
 
 ```bash
-cd /path/to/prestashop
-composer install
+$ cd /path/to/prestashop
+$ composer install
 ```
 
 ## Setting up file rights
 
 PrestaShop needs recursive write permissions on several directories:
 
-- /admin-dev/autoupgrade/
-- /app/logs
-- /app/Resources/translations
-- /cache
-- /config/themes
-- /download
-- /img
-- /log
-- /mails
-- /modules
-- /themes
-- /translations
-- /var
+- ./admin-dev/autoupgrade
+- ./app/logs
+- ./app/Resources/translations
+- ./cache
+- ./config/themes
+- ./download
+- ./img
+- ./log
+- ./mails
+- ./modules
+- ./themes
+- ./translations
+- ./var
+
+You can setup proper permissions with that one line:
+```bash
+$ sudo chmod +w -R admin-dev/autoupgrade app/logs app/Resources/translations cache config/themes download img log mails modules themes translations var
```

Comment:
Hi @LocalHeroPro thanks for the suggestion however this only works on specific environment. Probably most of us work on such an environment, but we cannot exclude those of us who work on specific distros or event ... Windows 😭

Reply:
Also it's maybe not the best practice to use `sudo` 😉

Reply:
So maybe prepare those steps and add EXTRA heading, that is for Linux environment? After that everyone - developers - can more friendly reproduce steps for their own environment.

If no `sudo` so what is better?

Reply:
I'm working on Linux and never had to run this kind of command to make my apache / Nginx works with PrestaShop :thinking: 
FYI: if you have a fresh install of Debian, sudo is not installed.

Reply:
So, I have to remove `sudo` and everything is fine for you gays?

Reply:
I think it would be good without sudo :)